README.md

```diff
@@ -42,6 +51,39 @@ Note: when adding the plugin's javascript to your web page, include the
 Video.js. The plugin's built javascript file expects there to be a reference to Video.js
 at `window.videojs` and will throw an error if it does not exist.
 
+### Initialization options
+
+* **`preloadWebComponents`** (default: `false`) - The Chromecast framework relies on the
+  `webcomponents.js` polyfill when a browser does not have `document.registerElement`.
+  If you are using jQuery, this polyfill must be loaded and initialized before jQuery is
+  initialized. Unfortunately, the Chromecast frameworks loads the `webcomponents.js`
+  polyfill via a dynamically created `<script>` tag. This causes a race condition (see
+  #17). Setting `preloadWebComponents` to `true` will make this plugin add the
+  `webcomponents.js` polyfill synchronously when the polyfill is needed.
```

Comment:
Should add a bit more to this paragraph to explain what that means. For example:

> If you use the `preloadWebComponents: true` option, you should make sure that this plugin is loaded before jQuery. Then include the framework after this plugin as you normally would.
>
> tl;dr: if you use jQuery, you should use the `preloadWebComponents: true` option in this plugin.